.github/workflows/test.yml

```diff
@@ -81,6 +81,13 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - run: |
+        sudo cp /etc/docker/daemon.json /tmp/daemon.json.tmp
+        sudo truncate -s 0 /etc/docker/daemon.json
+        jq '. |= . + { "ipv6": true, "fixed-cidr-v6": "fd00::/64" }' /tmp/daemon.json.tmp | sudo  tee -a /etc/docker/daemon.json
+        sudo rm /tmp/daemon.json.tmp
```

Comment:
I suggest using sponge

https://stackoverflow.com/questions/36565295/jq-to-replace-text-directly-on-file-like-sed-i